scipy/stats/_continuous_distns.py

```diff
@@ -4192,11 +4196,8 @@ def _entropy(self):
         estimation of the Laplace distribution parameters, so the keyword
         arguments `loc`, `scale`, and `optimizer` are ignored.\n\n""")
     def fit(self, data, *args, **kwds):
-        floc = kwds.pop('floc', None)
-        fscale = kwds.pop('fscale', None)
-
-        _check_fit_input_parameters(data, args,
-                                    kwds, fixed_param=(floc, fscale))
+        data, floc, fscale = _check_fit_input_parameters(self, data,
+                                                         args, kwds)
```

Comment:
While we're here, just a touch-up below to mirror what we settled on for Pareto.

```
        # Source: Statistical Distributions, 3rd Edition. Evans, Hastings,
        # and Peacock (2000), Page 124

        if floc is None:
            floc = np.median(data)

        if fscale is None:
            fscale = (np.sum(np.abs(data - loc))) / len(data)

        return floc, fscale
```